File: php-laravel/lib/app/Models/ResponseRoutesResult.php
<?php
/**
 * ResponseRoutesResult
 */
namespace app\Models;

/**
 * ResponseRoutesResult
 */
class ResponseRoutesResult {

    /** @var string $searchId */
    private $searchId;

    /** @var \app\Models\ResponseRoutesLocation[] $locations */
    private $locations;

    /** @var string[] $unreachable */
    private $unreachable;

}

File: kotlin-spring/src/main/kotlin/traveltimeplatform/models/ResponseRoutesResult.kt
package traveltimeplatform.models

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import traveltimeplatform.models.ResponseRoutesLocation
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param searchId 
 * @param locations 
 * @param unreachable 
 */
data class ResponseRoutesResult (

        @get:NotNull 
        @JsonProperty("search_id") val searchId: String,

        @get:NotNull 
        @JsonProperty("locations") val locations: List<ResponseRoutesLocation>,

        @get:NotNull 
        @JsonProperty("unreachable") val unreachable: List<String>
) {

}

